src/services/index.ts

```diff
@@ -1,3 +1,5 @@
 // SPDX-FileCopyrightText: 2024 PNED G.I.E.
 //
 // SPDX-License-Identifier: Apache-2.0
+import CKAN from './ckanapi';
+export { CKAN };
```

Comment:
I am not familiar with React, is `index.ts` required to make the services discoverable for IoC / dependency injection?

Reply:
it is a common practice in javascript world that when you have a lot of files you need to export from a folder, then having index file makes it easy during import without having to expose which file contains which export

so you could write

`import { CKAN } from './services';`
 `import { XYZ } from './services`

instead of 

`import { CKAN } from './services/ckanapi';`
 `import { XYZ } from './services/xyz`